inclusive-design/wecount.inclusivedesign.ca
# Add YouTube Icon to Footer (resolves #356)

## Description
This PR adds the YouTube Icon to the footer and links to the We Count YouTube channel.


## Steps to test

1. Go to deploy preview
2. Scroll down to footer
3. Observe and click on YouTube Icon

**Expected behavior:** 
YouTube icon should correspond to the changes for UIO settings and clicking on the icon should navigate to the We Count YouTube Channel.

## Additional information
N/A


## Related issues
closes #356
